pkgs/development/libraries/sqlite/default.nix

```diff
@@ -53,6 +54,9 @@ stdenv.mkDerivation rec {
 
     # https://sqlite.org/src/info/2025-02-16T10:57z
     ./CVE-2025-3277_CVE-2025-29087.patch
+
+    # https://www.sqlite.org/src/info/5508b56fd24016c13981ec280ecdd833007c9d8dd595edb295b984c2b487b5c8
+    ./CVE-2025-6965.patch
```

Comment:
Why does the patch differ from the posted URL?  In particular, the `src/sqliteInt.h` change seems missing.

Reply:
The patch differs because we are not building sqlite3 from the "raw" sources but from the "amalgamation": https://sqlite.org/amalgamation.html


The macros do not exist in this version they were added later on https://www.sqlite.org/src/info/215650a5a1d55bdb
That's said we probably should not leave assertions relying on it. I'm going to adjust the patch.

Reply:
Ah, the assertions only compiled because we skip them in preprocessor already.